engine/src/juliabox/db/instance.py

```diff
@@ -99,7 +99,12 @@ def get_stale_instances(cluster):
         valid_time = nowsecs - JBoxInstanceProps.SESS_UPDATE_INTERVAL
         stale = []
         for record in JBoxInstanceProps.scan(instance_id__beginswith=cluster, publish_time__lt=valid_time):
```

Comment:
Looks like it may be simpler to scan for records beginning with `JBoxDB.qual(cluster, '')`. We won't need the `find` below then as it will pick up only valid entries.